prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/FhirPathUtils.kt

```diff
@@ -171,7 +174,7 @@ object FhirPathUtils : Logging {
      * Convert a FHIR [dateTime] to the format required by HL7
      * @return the converted HL7 DTM
      */
-    fun convertDateTimeToHL7(dateTime: BaseDateTimeType): String {
+    fun convertDateTimeToHL7(dateTime: BaseDateTimeType, appContext: CustomContext? = null): String {
```

Comment:
I don't see this function getting called with the customContext passed in. Can you look for a way to extract this logic out of `FhirPathUtils`. We don't want to have ReportStream logic inside the conversion library. Maybe there's a way to pass this function through the CustomContext.